apps/right-to-rent-check/acceptance/features/person-in-property.js

```diff
@@ -21,15 +21,15 @@ Scenario('I see the correct fields on the page', (
   ]);
 });
 
-Scenario('I see errors if I submit a form without selecting anything', (
+Scenario('WHEN I submit a form without selecting anything THEN I see errors', (
```

Comment:
Consider `When I do not select a field And I submit the form Then I see an error`